src/types/documentProducer/Variable.ts

```diff
@@ -117,6 +117,15 @@ export const VariableStatuses: VariableStatusType[] = [
   'Rejected',
 ];
 
+export const NonSectionVariableStatuses: VariableStatusType[] = [
+  'Approved',
```

Comment:
I didn't see this used, is that for a later PR?

Reply:
Oh, thanks. I forgot to push one file